server/src/main/java/org/elasticsearch/action/admin/cluster/repositories/verify/VerifyRepositoryResponse.java

```diff
@@ -45,31 +133,33 @@
 
     public VerifyRepositoryResponse(ClusterName clusterName, DiscoveryNode[] nodes) {
         this.clusterName = clusterName;
-        this.nodes = nodes;
+        this.nodes = Arrays.asList(nodes);
     }
 
     @Override
     public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
-        clusterName = new ClusterName(in);
-        nodes = new DiscoveryNode[in.readVInt()];
-        for (int i=0; i<nodes.length; i++){
-            nodes[i] = new DiscoveryNode(in);
+        if (in.getVersion().onOrAfter(Version.V_6_4_0)) {
```

Comment:
Typically when we make a change like this in master we would set the version to 7.0.0 as otherwise it will be impossible to have a green CI build here since 6.x (6.4) does not have this code yet. Then when you backport you will flip the version to 6.4.0 and push a commit to master to change the version to 6.4.0.

Reply:
oh i wonder if this is why my bwc tests are failing!

Reply:
Almost surely. 😇